int32 robot
multirob_test/cmdPickup pickup
bool canDo
int32 distance
int32 space
